bulma-rtl.sass

```diff
@@ -0,0 +1,10 @@
+/*! bulma.io v0.4.0 | MIT License | github.com/jgthms/bulma */
+@charset "utf-8"
+
+$right: left
+$left: right
+
+html
+  direction: rtl
+
+@import "bulma"
```

Comment:
this is the way many css libs out there releasing RTL versions. generally we cannot have the smart BIDI behavior only with css and the BIDI is only applicable we the powers of js. so with css only approach we have 2 types of implementations:
* making a separate RTL version which will have the same KBs in size but it's standalone and there is no need to include the LTR version too. (my approach is this)
* another way to handle that is making an override css file so we should include LTR version in page and then override styles for RTL with another file. which I see is more dirty than first approach since you need to include bulma.css and bulma-rtl.css both in page. also overriding the styles are not optimal (computational and rendering wise).